lib/child_process.js

```diff
@@ -600,7 +600,7 @@ function abortChildProcess(child, killSignal) {
 
 function spawn(file, args, options) {
   options = normalizeSpawnArguments(file, args, options);
-  validateTimeout(options.timeout, 'options.timeout');
+  validateTimeout(options.timeout);
```

Comment:
Hmm.. should we alternatively modify the validator function to use the name?

Reply:
I think that is a good idea.

Reply:
This is a validator function defined and used only in this file. If we add that argument, it would always be the same string literal, `'options.timeout'`. It never validates anything that isn't `options.timeout`. I'd prefer to leave it alone unless/until that becomes a feature it needs.

I will note that it currently reports this as the error message:

> The value of "timeout" is out of range. It must be an unsigned integer.

I think that's not at all a problem as the user will know they sent a "timeout" property (and may not always know what "options" refers to without looking at the docs). But if we want to update that to say "options.timeout", let's just update it inside `validateTimeout()` and not add a basically unused second argument. YAGNI and all that.